examples/todomvc/components/App.vue

```diff
@@ -13,10 +13,11 @@
     </header>
     <!-- main section -->
     <section class="main" v-show="todos.length">
-      <input class="toggle-all"
+      <input class="toggle-all" id="toggle-all"
```

Comment:
I see, yeah the `+` doesn't leave us other option here (at least none that I'm aware of)